### PR TITLE
niclake - Keymap updates - Boardwalk updates & DZ60 create

### PR DESCRIPTION
**Pull request by niclake**
## Description

Adding my DZ60 keymap, and updating my Boardwalk keymap

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/boardwalk/keymaps/niclake/keymap.c [hunk 1]
```diff
@@ -37,7 +37,7 @@ enum custom_keycodes {
 #define FN MO(_FN)
 #define NUM TT(_NUM)
 #define ADJ MO(_ADJ)
-#define MACWIN AG_TOGG
+#define MACWIN MAGIC_TOGGLE_ALT_GUI
```
Comment (drashna) [inline keyboards/boardwalk/keymaps/niclake/keymap.c:40]: This is the same keycode, in theory.

File: keyboards/dz60/keymaps/niclake/keymap.c [hunk 2]
```diff
@@ -0,0 +1,67 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/dz60/keymaps/niclake/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/boardwalk/keymaps/niclake/keymap.c (hunk 1 above)
Comment (niclake) [reply keyboards/boardwalk/keymaps/niclake/keymap.c:40]: I know, this was for my own well-being of being able to remember why I have `BOOTMAGIC_ENABLE` turned on (as it's easier for me to search for across my project). I only changed my own define here.

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**